views/js/chatControl.js

```diff
@@ -27,6 +27,10 @@ const sanitizeHTML = (html) => {
 const postMessageToChat = () => {
   const name = document.getElementById("name_input").value;
   const message = document.getElementById("message_input").value;
+  if (!message) {
+    alert('Please enter a message.');
```

Comment:
It's not necessary per say, I just wanted to notify the user if they try to send a chat message without an actual message (e.g. they accidentally enter the message in the name field instead of the message box). Do you think it should silently return?

Reply:
I don't know -- alerts tend to be fairly modal, but that may not be a big issue in this case.